modules/ROOT/pages/manage-message-flows.adoc

```diff
@@ -31,19 +31,23 @@ You can also undeploy or delete message flows entirely if your partner relations
 * xref:endpoint-as2-receive.adoc[AS2 Endpoint Settings]
 
 * xref:endpoint-sftp-receive-target.adoc[SFTP Receive Endpoint Settings]
-
-. If you want to upload a different DataWeave map, click *Import* from the Map section to choose a different DataWeave map and click *Upload mapping*.
-
++
+. If you want to replace your existing DataWeave map, click *Import* from the Map section to choose a different DataWeave map and click *Upload mapping*.
++
+** If you want to examine the map, you can download the `.dwl` file by clicking its name link.
```

Comment:
the map before you import it?
Is a user likely to do this right in the middle of this other task?
```suggestion
** If you want to examine the map, you can download it by clicking its <name>.dwl link.
```

Reply:
It is most likely that the user will download an existing map to examine it. This is because something needs to change on it. So, it would come after uploading.